tests/test_nestedattr.py

```diff
@@ -0,0 +1,44 @@
+from datetime import date
+from pprint import pprint
```

Comment:
major: unused imports

Reply:
`from datetime import date` is used twice on `release_date=date(1971, 12, 17)`

Reply:
meant `pprint` which is the commented line :)